app/webpacker/src/stylesheets/_rules_of_origin.scss

```diff
@@ -37,3 +37,13 @@
     margin: 0 1em 0 0;
   }
 }
+
+.rules-of-origin-met-message {
+  padding: govuk-spacing(3);
+  background-color: govuk-colour("light-grey");
+  margin-bottom: 2em;
+
+  p {
+    margin-bottom: 0px;
```

Comment:
I _think_ the correct syntax is just `0` rather then `0px` since the unit is redundant, ie `0px == 0rem == 0em == 0vh`